code/datums/gamemode/role/ninja.dm

```diff
@@ -727,7 +761,7 @@ Suit and assorted
 	to_chat(user, "[message]")
 
 /obj/item/weapon/katana/hesfast/suicide_act(mob/user)
-	visible_message("<span class='danger'>[user] is slicing \his chest open with the [src.name]! It looks like \he's trying to commit sudoku.</span>")
+	visible_message("<span class='danger'>[user] is slicing \his chest open with the [src.name]! It looks like \he's trying to commit [pick("seppuku","sudoku","harikari","crossword puzzle")].</span>")
```

Comment:
This one specifically aims at the chest to commit sudoku, unlike the normal one which is aimed at the belly. Don't feel like this works.